marlowe-dashboard-client/src/Capability/Marlowe.purs

```diff
@@ -295,22 +296,10 @@ instance manageMarloweAppM :: ManageMarlowe AppM where
           Right observableState -> Right (plutusAppId /\ observableState)
   subscribeToPlutusApp = toBack >>> Left >>> Subscribe >>> sendWsMessage
   subscribeToWallet =
-    toBack
-      >>> un Back.Wallet
-      >>> _.getWalletId
-      >>> (\x -> Back.PubKeyHash { getPubKeyHash: x })
-      >>> Right
-      >>> Subscribe
-      >>> sendWsMessage
+    sendWsMessage <<< Subscribe <<< Right <<< WI.toPubKeyHash
```

Comment:
This uses the toPubKeyHash which is incorrect, but I noticed that `sendWsMessage` is using `CombinedWSStreamToServer` which is defined in the PAB and exported as 

```purescript
data CombinedWSStreamToServer
  = Subscribe (Either ContractInstanceId PubKeyHash)
  | Unsubscribe (Either ContractInstanceId PubKeyHash)
```
I think that type is wrong and the PubKeyHash should be changed to WalletId. But that is out of the scope of this PR.